packages/components/context-menu/index.js

```diff
@@ -168,7 +170,9 @@ class ContextMenu extends Component {
         this.menuRef.current.style.minWidth = "210px";
       }
 
-      this.menuRef.current.style.left = left + "px";
+      if (this.props.theme.interfaceDirection === "rtl")
```

Comment:
Remove the condition on the direction of the interface, if the style ( this.menuRef.current.style.left) does not depend on the direction.

Reply:
deleted